.gitignore

```diff
@@ -8,6 +8,7 @@ _site
 *.sublime-*
 _site
 dist/*.js
+dist/*.min.css
```

Comment:
I'm not trying to complicate things, but now there are both non-`.min` (`leaflet.js`) and `.min` (`leaflet.min.css`) files that are minified/uglified. I wonder:

- If the minified CSS is `leaflet.min.css`, should all instances of `leaflet.css`  be changed to `leaflet.min.css` in [download.md](https://github.com/Leaflet/Leaflet/blob/918b65f1cc2dea81dbdfa67b676602524c57ee23/docs/SlavaUkraini/download.md?plain=1) (and maybe other places)?
- Or, should there be a `src/leaflet.css` to output the minified version as `dist/leaflet.css`? I think this has the benefit that there's no need to change references from `leaflet.css` to `leaflet.min.css`.

Reply:
This is a valid comment, I chose to keep the PR simple for now to start this discussion. We could certainly change these references if that would make sense.

We could also first land this change and then start a discussion on if and how we further integrate it.

Reply:
To clarify: per https://github.com/Leaflet/Leaflet/pull/8123#issuecomment-1091610570, the idea is now to rename the current `dist/leaflet.css` file to `dist/leaflet-src.css` and generate the minified `dist/leaflet.css` (and not change any references in the docs).

Reply:
Done :+1: